data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/consumer/KafkaSourceCustomConsumer.java

```diff
@@ -117,17 +157,13 @@ public void shutdownConsumer(){
     }
     @Override
     public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
-        for (TopicPartition partition : partitions) {
-            consumer.seek(partition, lastReadOffset);
+        for (TopicPartition topicPartition : partitions) {
+            Long committedOffset = consumer.committed(topicPartition).offset();
+            consumer.seek(topicPartition, committedOffset);
         }
     }
 
     @Override
     public void onPartitionsRevoked(Collection<TopicPartition> partitions) {
-        try {
```

Comment:
This is now an empty method

Reply:
Yeah. Want to keep it for any future actions to be taken in case of partitions revoked.